docs/installation.rst

```diff
@@ -129,11 +137,21 @@ You can follow the steps below to compile FlashInfer from source code:
 
 4. Compile FlashInfer:
 
-   .. code-block:: bash
+   .. tabs::
+
+       .. tab:: JIT mode
+
+           .. code-block:: bash
+
+               cd flashinfer/python
+               pip install -e . -v
+
+       .. tab:: AOT mode
 
-       cd flashinfer
-       pip install -e . -v
+           .. code-block:: bash
 
+               cd flashinfer/flashinfer-aot
```

Comment:
Will we release AOT and JIT wheel together?

Reply:
Yes, JIT wheels on pypi and AOT wheels on self-hosted index.